AzureResourceManager/Artifactory/MP_submission/scripts/install_artifactory7.sh

```diff
@@ -111,7 +121,44 @@ server {
     }
 }
 EOF
+else
+
+cat <<EOF >/etc/nginx/conf.d/artifactory.conf
+## server configuration
+server {
+  listen 80 ;
+  server_name ~(?<repo>.+)\\.${CERTIFICATE_DOMAIN} artifactory ${ARTIFACTORY_SERVER_NAME}.${CERTIFICATE_DOMAIN};
```

Comment:
this should be
server_name _;
This matches all servers so we do not have to worry about domains etc.